W-20223234 connector builder mcp tools (#14)
mulesoft/docs-mulesoft-mcp-server: modules/ROOT/pages/getting-started.adoc

Learn how to set up and configure MuleSoft MCP Server for your agent-ready IDE. 

[IMPORTANT] 

> If you have Anypoint Code Builder or Connector Builder, stop here. Don't perform these steps. MuleSoft MCP Server is already installed and configured in MuleSoft Dev Agent in Anypoint Code Builder, or in Connector Builder. Authentication happens through the user that's logged into Anypoint Code Builder or Connector Builder. MuleSoft Dev Agent assumes any permissions that the user has. For more information about MuleSoft Dev Agent or Connector Builder, see xref:anypoint-code-builder::api-ai-create-spec.adoc[Creating API Specs with MuleSoft Dev Agent] or xref:connector-builder::index.adoc[].

Reviewer: ```suggestion
====
If you have Anypoint Code Builder or Connector Builder, stop here. Don't perform these steps. 

MuleSoft MCP Server is already installed and configured in MuleSoft Dev Agent in Anypoint Code Builder, and Connector Builder. Authentication happens via the logged-in user and MuleSoft Dev Agent assumes any permissions that the user has.

For more information about MuleSoft Dev Agent and Connector Builder, see xref:anypoint-code-builder::api-ai-create-spec.adoc[] and xref:connector-builder::index.adoc[].
====
```
